core/src/main/java/hudson/slaves/ChannelPinger.java

```diff
@@ -136,6 +150,7 @@ public void onClosed(Channel channel, IOException cause) {
         });
 
         t.start();
-        LOGGER.fine("Ping thread started for " + channel + " with a " + interval + " minute interval");
+        LOGGER.fine("Ping thread started for " + channel + " with a " +
+            intervalSeconds + " second interval and a " + timeoutSeconds + " second timeout.");
```

Comment:
SC will be worse - as it still involves garbage collection for the implicit StringBuilder and Object/array creation is highly optimized
(I pine for slf4j....)